collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -296,12 +296,8 @@ public void onClick(View v) {
                 ToastUtils.showLongToast(R.string.corrupt_settings_file_notification);
             }
         }
-    }
 
-    private void initToolbar() {
-        Toolbar toolbar = findViewById(R.id.toolbar);
-        setTitle(getString(R.string.main_menu));
-        setSupportActionBar(toolbar);
+        setUpStorageMigrationBanner();
```

Comment:
I don't think this is needed. Because of the `KEYS_WE_SHOULD_NOT_RESET` map, the scoped storage setting can't be changed by a settings file.

Reply:
It's called in onResume() and that's enough.

Reply:
Oh my bad another mistake. I was experimenting with test coverage and left this in. @grzesiek2010 is totally right it's only needed in `onResume`